src/app/legacy/containers/CpsRecommendations/index.jsx

```diff
@@ -46,33 +46,29 @@ const LabelComponent = styled(SectionLabel)`
 `;
 
 const CpsRecommendations = ({ items }) => {
-  const { recommendations, translations, script, service, dir } =
-    useContext(ServiceContext);
+  const { recommendations, script, service, dir } = useContext(ServiceContext);
   const { enabled } = useToggle('cpsRecommendations');
 
+  const {
+    palette: { GREY_2 },
+  } = useTheme();
+
+  const title = useWsojTitle({ variation: undefined });
```

Comment:
Can we remove this (if it's defaulted to undefined anyway?)
```suggestion
  const title = useWsojTitle();
```